addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1042,7 +1042,12 @@ const Wysiwyg = Widget.extend({
                     if ($node.hasClass('o_editable_date_field_format_changed')) {
                         $nodes.addClass('o_editable_date_field_format_changed');
                     }
-                    $nodes.html($node.html());
+                    for (const node of $nodes) {
+                        const html = $node.html()
```

Comment:
Why do that inside the loop tho? It should be the same across the entire iteration no?

Reply:
You are indeed correct.

Reply:
@Goaman